ci/docker/php-fpm/Dockerfile

```diff
@@ -5,12 +5,15 @@ COPY docker/php-fpm/app.ini /usr/local/etc/php/conf.d/
 # Copy the app_test.php.dist to web/app.php
 COPY index.php ../public/index.php
 
+# Yank the node and npm binaries from the official Node docker container
+COPY --from=node:14 /usr/local/lib/node_modules /usr/local/lib/node_modules
+COPY --from=node:14 /usr/local/bin/node /usr/local/bin/node
+RUN ln -s /usr/local/lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm
+RUN npm install -g npx
```

Comment:
Why apt install when we can get it from the official node docker container..

This is what we did in the EngineBlock Github Actions Test integration. And it works quite well in my opinion.